ext/crypto/export_key.rs

```diff
@@ -168,3 +189,156 @@ fn export_key_rsa(
     }
   }
 }
+
+fn bytes_to_b64(bytes: Vec<u8>) -> String {
+  base64::encode_config(bytes, base64::URL_SAFE_NO_PAD)
+}
+
+fn export_key_ec(
+  format: ExportKeyFormat,
+  key_data: RawKeyData,
+  algorithm: ExportKeyAlgorithm,
+  named_curve: EcNamedCurve,
+) -> Result<ExportKeyResult, deno_core::anyhow::Error> {
+  match format {
+    ExportKeyFormat::Spki => {
+      let subject_public_key = match named_curve {
+        EcNamedCurve::P256 => {
+          let point = key_data.as_ec_public_key_p256()?;
+
+          point.as_ref().to_vec()
+        }
+        EcNamedCurve::P384 => {
+          let point = key_data.as_ec_public_key_p384()?;
+
+          point.as_ref().to_vec()
+        }
+        _ => return Err(data_error("Unsupported named curve")),
+      };
+
+      let alg_id = match named_curve {
+        EcNamedCurve::P256 => <p256::NistP256 as p256::elliptic_curve::AlgorithmParameters>::algorithm_identifier(),
+        EcNamedCurve::P384 => <p384::NistP384 as p384::elliptic_curve::AlgorithmParameters>::algorithm_identifier(),
+        _ => return Err(data_error("Unsupported named curve"))
+      };
+
+      let alg_id = match algorithm {
+        ExportKeyAlgorithm::Ecdh { .. } => AlgorithmIdentifier {
+          oid: ObjectIdentifier::new("1.3.132.1.12"),
+          parameters: alg_id.parameters,
+        },
+        _ => alg_id,
+      };
+
+      // the SPKI structure
+      let key_info = spki::SubjectPublicKeyInfo {
+        algorithm: alg_id,
+        subject_public_key: &subject_public_key,
+      };
+
+      let spki_der = key_info.to_vec().unwrap();
+
+      Ok(ExportKeyResult::Spki(spki_der.into()))
+    }
+    ExportKeyFormat::Pkcs8 => {
+      // private_key is a PKCS#8 DER-encoded private key
+      let private_key = key_data.as_ec_private_key()?;
+
+      Ok(ExportKeyResult::Pkcs8(private_key.to_vec().into()))
+    }
+    ExportKeyFormat::JwkPublic => match named_curve {
+      EcNamedCurve::P256 => {
+        let point = key_data.as_ec_public_key_p256()?;
+        let coords = point.coordinates();
+
+        if let p256::elliptic_curve::sec1::Coordinates::Uncompressed { x, y } =
+          coords
+        {
+          Ok(ExportKeyResult::JwkPublicEc {
+            x: bytes_to_b64(x.to_vec()),
+            y: bytes_to_b64(y.to_vec()),
```

Comment:
```suggestion
            x: bytes_to_b64(x),
            y: bytes_to_b64(y),
```
Ditto for other calls that involve `GenericArray<u8, N>`